### DISCUSSION TITLE
Is there a way to set column width ?

### DISCUSSION THREAD

**Discussion by marukop** [miachm/SODS#68] [Q&A] [answered]
Hello,

I'm trying to set the width of a Range (or Style), but I don't see any means to achieve this in the Javadoc :/
Is this planned in a future release or is this achiveable by another mean ?

Regards

**Comment by miachm** [comment, marked as answer]
@marukop Columns/Rows size are managed in the Sheet class.

**Reply by marukop** [reply]
Thank you very much for your reply and sorry for the disturbance -_-   
I just saw the [method in the javadoc](https://miachm.github.io/SODS/com/github/miachm/sods/Sheet.html#setColumnWidth-int-java.lang.Double-).
